base_layer/wallet/src/transaction_service/protocols/transaction_receive_protocol.rs

```diff
@@ -258,7 +261,7 @@ where TBackend: TransactionBackend + Clone + 'static
                 .map_err(|e| TransactionServiceProtocolError::new(self.id, TransactionServiceError::from(e)))?;
 
             if !direct_send_result && !store_and_forward_send_result {
-                warn!(
+                error!(
```

Comment:
If this is retried again later, Id say warn! is correct?

Reply:
So this one terminates this protocol, it won't be retried later unless its done explicitly

Reply:
and right now there isn't actually an explicit way to do that so this is def an error because it shouldn't occur. If we received a Tx message then we should be able to send a reply via one of the routes.